Tests/Task2.robot

```diff
@@ -0,0 +1,62 @@
+*** Settings ***
+Library  SeleniumLibrary
+Resource  ../Resources/openweathermap.robot
+Resource  ../Resources/variables.robot
+
+
+Test Setup    Open Login Page    ${URL}
+Test Teardown    Close Browser
+
+
+*** Keywords ***
+User is redirected to sign in page on attempt to open home page
+    [Arguments]    ${HOME_URL}    ${SIGNIN_URL}
+    Go to    ${HOME_URL}
+    Sleep    5s
```

Comment:
sleep it's a bad approach to wait something as it does not guarantee that event happened after this time and in opposite if event happened pretty fast we just waste test time 

use Explicit Wait instead to wait for a specific event (it this case will be good to use `Wait Until Location Contains` from SeleniumLibrary)